scripts/build-all-bottles.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env nix-shell
+#! nix-shell shell.nix -i bash
+
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+if [ -z "$1" ] ; then
+    echo "Please call this script with the name of the OS for which to build the bottles."
+    exit 1
+fi
+
+# we don't bottle meta-formulas that contain only services
+formulae=("tezos-accuser-011-PtHangz2" "tezos-accuser-012-Psithaca" "tezos-admin-client" "tezos-baker-011-PtHangz2" "tezos-baker-012-Psithaca" "tezos-client" "tezos-codec" "tezos-endorser-011-PtHangz2" "tezos-node" "tezos-sandbox" "tezos-signer")
+
+# tezos-sapling-params is used as a dependency for some of the formulas
+# so we handle it separately.
+# We don't build the bottle for it because it is never updated over time.
+brew install --formula ./Formula/tezos-sapling-params.rb
+
+for f in "${formulae[@]}"; do
+  # check if the formula doesn't already have a bottle in its respective release
+  if ./scripts/check-bottle-built.sh "$f" "$1"; then
+    # build a bottle
+    if ./scripts/build-one-bottle.sh "$f"; then
+      # upload the bottle to its respective release
+      FORMULA_TAG=$(sed -n "s/^\s\+version \"\(.*\)\"/\1/p" "./Formula/$f.rb")
+      export FORMULA_TAG
```

Comment:
btw, do we actually need to export this variable now?

Reply:
You're right, we don't